docs/agents/suprsync.rst

```diff
@@ -0,0 +1,184 @@
+.. highlight:: rst
+
+.. _suprsync:
+
+==============
+SupRsync Agent
+==============
+
+The SupRsync agent keeps a local directory synced with a remote server.
+It continuously copies over new files to its destination, verifying the copy
+by checking the md5sum and deleting the local files after a specified amount
+of time if the local and remote checksums match.
+
+.. argparse::
+    :filename: ../agents/suprsync/suprsync.py
+    :func: make_parser
+    :prog: python3 suprsync.py
+
+Copying Files
+-----------------
+
+The SupRsync agent works by monitoring a table of files to be copied in a
+sqlite database. The full table is :ref:`described here<SupRsyncFiles>`,
+but importantly it contains information such as the full local path, the
+remote path relative to a base directory determined by the SupRsync cfg,
+checksumming info, several timestamps, and an "archive" name which is used to
+determine whic SupRsync agent should manage each file. This makes it possible
+for  multiple SupRsync agents to monitor the same db, but write their archives
+to different base-directories or remote computers.
+
+.. note:: 
+
+   This agent does not remove empty directories, since I can't think of a
+   foolproof way to determine whether or not more files will be written to a
+   given directory, and pysmurf output directories will likely have
+   unregistered files that stick around even after copied files have been
+   removed. I think it's probably best to have a separate cronjob make that
+   determination and remove directory husks.
```

Comment:
1. No, pysmurf only checks once when the instance is created, so if you delete the current output directory it will break things.
2. Yeah, I think just verifying that the directories are old enough or just that you're not deleting the most recent outputs directory for a given slot in case that's still in use.

Reply:
Gotcha. How old is old enough to be deleted? I'm thinking of Katie's comments on needing certain files for the system to work, so they can't be moved off/deleted too soon.